examples/ssx-notepad-datavault/package.json

```diff
@@ -5,6 +5,7 @@
     "private": true,
     "main": "src/index.ts",
     "scripts": {
+        "start": "yarn dev",
```

Comment:
ssx-test-dapp is working, but ssx-notepad-datavault isn't

Reply:
Fixed. Made dataVaults default off, and forgot to enable.